source/_docs/configuration/templating.markdown

```diff
@@ -215,6 +216,8 @@ The same thing can also be expressed as a filter:
 - `is_device_attr(device_or_entity_id, attr_name, attr_value)` returns whether the value of `attr_name` for the given device or entity ID matches `attr_value`. Not supported in [limited templates](#limited-templates).
 - `device_id(entity_id)` returns the device ID for a given entity ID or device name. Can also be used as a filter.
 
+Device IDs are automatically generated when you add a new device, and may look like this: `deadbeefdeadbeefdeadbeefdeadbeef`.
```

Comment:
How is this relevant here?

Reply:
The `deadbeefdeadbeefdeadbeefdeadbeef` string is used in the examples below as an example device id. This looked very funny to me, so here's an explanation what this string means.

Reply:
Well, to me this whole line feels out of scope and out of the blue in the context of this page.